verl/trainer/config/rollout/rollout.yaml

```diff
@@ -31,10 +31,16 @@ gpu_memory_utilization: 0.5
 # Whether to ignore EOS and continue generating after EOS is hit.
 ignore_eos: False
 
-# Whether to disable CUDA graph. Default True to allow cache freeing.
-enforce_eager: True
+# Whether to disable CUDA graph. Default False to best performance.
+enforce_eager: False
```

Comment:
What if `enforce_eager=False` but  `cudagraph_capture_sizes` is null, is it still capture cuda graph?

Reply:
It will use vllm default capture sizes, which is [1,2,4...512].